cpp/util/etcd_delete.h

```diff
@@ -11,6 +11,10 @@
 namespace cert_trans {
 
 
+// Delete keys in batches (implemented using concurrent requests). The
+// "keys" argument are pairs of key and modified index. If the index
+// for an entry is zero, EtcdClient::ForceDelete will be used instead
+// of EtcdClient::Delete.
```

Comment:
Not super keen on the special casing here, how about a separate method `EtcdForceDeleteKeys()` ?